django/applications/catmaid/control/skeleton.py

```diff
@@ -1590,16 +1595,17 @@ def connectivity_matrix_csv(request, project_id):
     pseudo_buffer = Echo()
     writer = csv.writer(pseudo_buffer, quoting=csv.QUOTE_NONNUMERIC)
 
-    response = StreamingHttpResponse((writer.writerow(row) for row in csv_data),
-            content_type='text/csv')
+    response = StreamingHttpResponse((writer.writerow(row) for row in csv_data), # type: ignore
+            content_type='text/csv') # FIXME: writerow does not have a return; this is definitely a bug
```

Comment:
Testing `writerow()` in a Python shell makes it return the input in CSV format and the API endpoint does work on production systems (just tested it). I took it straight from the [official docs](https://docs.djangoproject.com/en/2.1/howto/outputting-csv/#streaming-large-csv-files).